msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -418,6 +418,49 @@ public async Task<AuthenticationResult> AcquireTokenAsync(IEnumerable<string> sc
                         behavior, extraQueryParameters, parent, ApiEvent.ApiIds.AcquireTokenWithScopeUserBehaviorAuthority).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Non-interactive request to acquire token via Kerberos / Windows Integrated Authentication.
+        /// The username for which the flow is executed is pulled from the operating system, as the current user principal name
+        /// </summary>
+        /// <param name="scopes">Array of scopes requested for resource</param>
+        /// <returns>Authentication result containing token of the current login user</returns>
```

Comment:
Authentication result containing a token for the requested scopes and for the currently logged-in user in Windows